### PR TITLE
Ofbiz 11451: add the license

### PR DESCRIPTION
**Pull request by PierreSmits**

### CODE REVIEW

**Comment by mbrohl** [comment]
Can you please remove the merge commits?

**Comment by jacopoc** [comment]
The license should not contain references to themes and framework since these are not part of the plugins.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+=========================================================================
+
+This product bundles "Noto Sans" fonts, which are available under the
+"Apache License Version 2.0".
+For details, see framework/resources/fonts/NotoSans/.
+
```
Comment (jacopoc) [inline LICENSE:213]: This section should be removed, because it references framework (which is not part of plugins).

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+This product bundles "timezones" files, which are available in the "Public Domain".
+For details, see themes/common/webapp/common/js/plugins/date/timezones.
+
+=========================================================================
+
+This product bundles "Date Format 1.2.3", which is available
+under the "MIT" license. 
+For details, see themes/common/webapp/common/js/plugins/date/date.format-1.2.3*
+
```
Comment (jacopoc) [inline LICENSE:230]: This section should be removed, because it references themes (which is not part of plugins).

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+=========================================================================
+
+This product bundles "jQuery" and "jQuery" plugins and extensions, which are available
+under the "MIT" license. For details, see:
+themes/common/webapp/common/js/jquery
```
Comment (jacopoc) [inline LICENSE:235]: This line should be removed.

**Comment by PierreSmits** [comment]
Closing due to contamination from merge commits. Will open a new one

**Pull request closed without merging**